Update README.md (#38)
raviriley/agency-jekyll-theme: README.md

**If you enjoy this theme, please consider [supporting me](https://www.paypal.me/raviriley) to continue developing and maintaining it.**

<div align="center">

[![Support via PayPal](https://cdn.rawgit.com/twolfson/paypal-github-button/1.0.0/dist/button.svg)](https://www.paypal.me/raviriley)

</div>

## Installation

There are three ways to install this theme:

1. As a gem-based theme
2. Use the [starter template][template] (best for GitHub Pages)
3. As a remote theme

#### 1. Gem-based Theme Installation

> Replace the contents of your `_config.yml` file with the sample [\_config.yml](https://github.com/raviriley/agency-jekyll-theme-starter/blob/master/_config.yml).

Author: ```suggestion
Replace the contents of your `_config.yml` file with the sample [\_config.yml](https://raw.githubusercontent.com/raviriley/agency-jekyll-theme-starter/master/_config.yml).
```

Install the gem with:

```sh
$ bundle add jekyll-agency
```

Or manually.

1. Add this line to your Jekyll site's `Gemfile`:
    ```ruby
    gem "jekyll-agency"
    ```
2. Then execute:
    ```sh
    $ bundle install
    ```

#### 2. Using the [Starter Template][template]

This is the fastest and easiest way to get up and running on GitHub Pages.

Simply generate your own repository by clicking the button below. Then replace the sample content with your own and configure for your needs.

<div align="center">

[![Use this template](https://img.shields.io/badge/Generate-Use_this_template-2ea44f?style=for-the-badge)][generate]

</div>
    
#### 3. Remote Theme Installation

> Replace the contents of your `_config.yml` file with [\_config.yml](https://github.com/raviriley/agency-jekyll-theme-starter/blob/master/_config.yml) and your `Gemfile` with [Gemfile](https://github.com/raviriley/agency-jekyll-theme-starter/blob/master/Gemfile.yml).

Author: ```suggestion
Replace your `_config.yml` file with the starter [\_config.yml](https://raw.githubusercontent.com/raviriley/agency-jekyll-theme-starter/master/_config.yml).

Replace your `Gemfile` with the starter [Gemfile](https://raw.githubusercontent.com/raviriley/agency-jekyll-theme-starter/master/Gemfile).
```

## Contributing

This project is intended to be a welcoming space for collaboration. If you have an idea, suggestion, feature request, etc., feel free to open an issue or pull request.

For bug reports, follow the provided template.

#### Improvements - Up for Grabs

- [ ] multiple language support (~~Spanish~~, Chinese, Arabic, etc.)
- [ ] customizable background coloring for each section

> - [x] ~~custom background images~~

Author: Made this a checkbox. Assuming it's done?
